access-grant/src/main/java/com/inrupt/client/accessgrant/AccessCredentialVerification.java

```diff
@@ -73,6 +82,37 @@ public List<String> getErrors() {
         return errors;
     }
 
+    /**
+     * Initialize the verification checks that were performed. This can only be called once, as the checks list is
+     * made unmodifiable.
+     *
+     * @param checks a list of any verification checks performed, never {@code null}
+     */
+    public void setChecks(final List<String> checks) {
+        this.checks = checks;
+        makeImmutable(this.checks);
```

Comment:
```suggestion
        this.checks = makeImmutable(checks);
```

All the setters should be adjusted similarly

Reply:
Fixed in e426ab78baf5781514d4c762597db99407cfceb3